src/libcore/iter/adapters/fuse.rs

```diff
@@ -37,35 +53,36 @@ where
 
     #[inline]
     default fn next(&mut self) -> Option<<I as Iterator>::Item> {
```

Comment:
Unrelated to the tweaks in this PR... there's a `default fn` here and elsewhere that shouldn't be (Specialization should be negotiated through internal traits.)

Reply:
Hmm, it's been like that for a _little_ while...
https://github.com/rust-lang/rust/pull/35656/files#diff-d93bf2799d551e25b17e877b23a37db3L1659-R1726

Reply:
Sure, we discovered problematic usage of associated type defaults much older, still, it needed to be fixed however old. :)

Reply:
@centril Maybe open an E-Easy issue for it with instructions for a new contributor to pick up?  I agree that it's "unrelated to the tweaks in this PR", so I don't think it needs to happen here.

Reply:
@scottmcm Oh sure, I didn't mean it should be fixed in *this* PR. :)

I've filed https://github.com/rust-lang/rust/issues/70796.